### PR TITLE
Task page

### PR DESCRIPTION
**Pull request by Mohitk391**
Added Task page

Files to check: 
1) Taskpage.js
2) Taskpage.css

preview:  https://62da860e4e159506c324ebfd--deluxe-macaron-ca250b.netlify.app/

### CODE REVIEW

File: src/pages/todo/Todo.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    height: 100px;
+    resize: none;
+}
+
+.time-input{
+    justify-content: space-evenly;
+}
+
+.time-input input{
+    width:70px;
```
Comment (abhisheksingh98) [inline src/pages/todo/Todo.css:115]: use rem instead of px

File: src/pages/todo/Todo.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+.task-contents input, #taskDescription{
+    border: none;
+    background-color: var(--bg-secondary);
+    outline:none;
+    border-radius: 10px;
+    padding: 5px 10px;
+}
+
+#taskDescription{
+    height: 100px;
```
Comment (abhisheksingh98) [inline src/pages/todo/Todo.css:106]: use rem instead of px

**Review by abhisheksingh98 — COMMENTED** [review]
Apart from px to rem changes, rest looks good to me. You can merge

File: src/pages/taskpage/Taskpage.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                                <i className="fa-solid fa-arrow-left-long"></i> TODO
+                                </button>
+                            </Link>
+                        </div>
+                        <div className="task-title">{task.name}</div>
+                        <div className="task-description">{task.description}</div>
+                    </div>
+                    <div className="tags">
+                        <h3>Tags:</h3>
+
```
Comment (varun-Tad) [inline src/pages/taskpage/Taskpage.js:71]: Can remove unnecessary space here

Comment (varun-Tad) [reply src/pages/taskpage/Taskpage.js:71]: Apart from the suggested changes it  looks good to merge

**Pull request merged**